upi/vsphere/README.md

```diff
@@ -33,15 +33,18 @@ The bootstrap ignition config must be placed in a location that will be accessib
 5. Ensure that you have you AWS profile set and a region specified. The installation will use create AWS route53 resources for routing to the OpenShift cluster.
 
 6. Run `terraform apply -auto-approve`.
+This will reserve IP addresses for the VMs.
+
+7. Run `terraform apply -auto-approve`.
 This will create the OpenShift cluster
```

Comment:
Why this two times?

Reply:
> The ip address query is still done by a data source,
> since there is no way to get output from a null resource. And because
> data sources are run before resources are created, the ip addresses
> are not available on the first terraform apply. The user needs to
> run terraform apply twice.